Strings/More.jsonc

```diff
@@ -59,65 +59,65 @@
 	"運転免許証の暗証番号・マイナンバーカードのパスワード": {
 		"ja": "運転免許証の暗証番号・マイナンバーカードのパスワード",
 		"en": "Driver's Licence & Individual Number Card (Japanese Only)",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "驾驶执照密码/个人号码卡 PIN 码（日本人适用）",
+		"zh-Hant": "駕駛執照密碼/個人號碼卡 PIN 碼（日本人適用）",
+		"zh-HK": "駕駛執照密碼/個人號碼卡 PIN 碼（日本人適用）"
 	},
 
 	"運転免許証の暗証番号の残り試行回数 および マイナンバーカードのパスワードの残り試行回数を確認することができます。": {
 		"ja": "運転免許証の暗証番号の残り試行回数 および マイナンバーカードのパスワードの残り試行回数を確認することができます。",
-		"en": "You can check the number of remaining attempts of your PIN/Password for your Driver's License/Individual Number Card.",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "You can check the number of remaining attempts of your PIN/Password for your Driver's Licence Card/Individual Number Card",
```

Comment:
You don't need to re-specify that it's a card here as it's defined at the end of the sentence. Also please return the period at the end of the sentence.